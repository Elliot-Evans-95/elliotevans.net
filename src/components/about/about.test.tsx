import * as React from "react";
import styled from '@emotion/styled';
import * as renderer from "react-test-renderer";

const Button = styled.div`
  color: hotpink;
`;

test("Button renders correctly", () => {
  expect(
    renderer.create(<Button>This is hotpink.</Button>).toJSON()
  ).toMatchSnapshot()
});
