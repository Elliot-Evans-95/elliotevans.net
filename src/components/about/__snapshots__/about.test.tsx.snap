// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button renders correctly 1`] = `
<div
  className="css-5p7k9v-Button eifzug00"
>
  This is hotpink.
</div>
`;
